src/pages/login/Login.jsx

```diff
@@ -56,100 +52,83 @@ const Login = () => {
       })
       .catch((err) => {
         console.error("Authentication Failed", err);
+        
         navigate("/login");
       });
   };
+
   return (
-    <div className="login__container_wrapper">
-      <div className="login__container">
-        <div className="top__header">
-          <div className="left__side__header">
-            <h2>Manager Login</h2>
-            <p>Make Sure Your Account is Secure</p>
-          </div>
-          <div className="right__side--header">
-            <img src={dckapPalliLogo} alt="dckap-logo" />
-          </div>
-        </div>
-        <div className="input__containers">
-          <div className="left__side__image__container">
-            <img src={managerLoginLogo} alt="manager-logo" />
+    <>
+      <div className="container">
+        <main className="login-page-container">
+          <div className="logo-container">
+            <img
+              className="logo"
+              src="/images/dckap_palli_logo_sm.svg"
+              alt=""
+              draggable={false}
+            />
           </div>
-          <div className="right__side-input__container">
-            <form
-              onSubmit={handleSubmit(handleLoginSubmit)}
-              className="login-form"
-            >
-              <div className={`email__inputs`}>
-                <label htmlFor={"email"}>
-                  email <span className="required-feild">*</span>
-                </label>
-                <div className={`email__icon`}>
-                  <span className={`material-symbols-outlined email-sybmbol`}>
-                    email
-                  </span>
-                  <div className="input__component">
-                    <input
-                      type={"text"}
-                      placeholder={`Enter the Email Id`}
-                      name={"email"}
-                      className={`email__feild`}
-                      {...register("email", {
-                        required: "Email is required",
-                        validate: isEmailValid,
-                      })}
-                    />
-                  </div>
+          <div className="login-container flex">
+            <div className="login-image-container">
+              <img src="/icons/Login Image.svg" alt="" draggable={false}/>
+            </div>
+            <div className="login-form-container">
+              <form onSubmit={handleSubmit(handleLogin)}>
```

Comment:
dont call function inside function handleSubmit(handleLogin) , i believe handleSubmit is function given by react-hook-form see how to extend that function and include your logic of handleLogin in docs